CHANGELOG.md

```diff
@@ -24,6 +24,7 @@
 - Support for URL path in host (#966)
 
 ### Changed
+- **Breaking:** default ors base path to be more dynamic ([#1036](https://github.com/VROOM-Project/vroom/issues/1036))
```

Comment:
Nit: I’d actually spell out that the hard coded /ors/v2 was removed and needs to be provided by in the argument. Saves a round trip to the PR to anyone reading this.

Reply:
Added this and moved it to ### Removed (more fitting)

Reply:
I'll just slightly rephrase that upon merging.